functions/index.js

```diff
@@ -1 +1,51 @@
+const fs = require('fs');
+const fetch = require('node-fetch');
+const path = require('path');
+const os = require('os');
 const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+
+const { databaseURL, storageBucket } = JSON.parse(process.env.FIREBASE_CONFIG);
+admin.initializeApp({
+  projectId: process.env.GCLOUD_PROJECT,
+  storageBucket: storageBucket,
+  databaseURL: databaseURL,
+});
+
+exports.onUserCreate = functions.firestore.document('/users/{userId}').onCreate(async (snap, context) => {
```

Comment:
Do you think also we should change the function name to something like `uploadProfilePicutreOnUserCreate`? or the name is clear enough?

Reply:
Since we might add more functionality to occur on user creation, I would think about extracting the contents of this function out, let's say to `reuploadUserPicture`, and then calling it from `onUserCreate`. Is it possible?